incubator/patroni/README.md

```diff
@@ -0,0 +1,81 @@
+$ helm install --name my-release patroni-x.x.x.tgz
+```
+
+## Configuration
+
+The following tables lists the configurable parameters of the patroni chart and their default values.
+
+|       Parameter         |           Description               |                         Default                     |
+|-------------------------|-------------------------------------|-----------------------------------------------------|
+| `Name`                  | Service name                        | `patroni`                                           |
+| `Namespace`             | Service namespace                   | `default`                                           |
+| `Spilo.Image`           | Container image name                | `registry.opensource.zalan.do/acid/spilo-9.5`       |
+| `Spilo.Version`         | Container image tag                 | `1.0-p5`                                            |
+| `ImagePullPolicy`       | Container pull policy               | `IfNotPresent`                                      |
+| `Replicas`              | k8s petset replicas                 | `5`                                                 |
+| `Component`             | k8s selector key                    | `patroni`                                           |
+| `Resources.Cpu`         | container requested cpu             | `100m`                                              |
+| `Resources.Memory`      | container requested memory          | `512Mi`                                             |
+| `Resources.Storage`     | Persistent volume size              | `1Gi`                                               |
+| `Credentials.Superuser` | password for the superuser          | `tea`                                               |
+| `Credentials.Admin`     | password for the admin user         | `cola`                                              |
+| `Credentials.Standby`   | password for the replication user   | `pinacolada`                                        |
+| `Etcd.Discovery`        | domain name of etcd cluster         | `<release-name>-etcd.<namespace>.svc.cluster.local` |
+| `GCS.Credentials`       | Google service account key file for authentication    | `<needs to be defined>`           |
+| `GCS.Bucket`            | GCS bucket name to stream WAL files and base backups  | `some-google-bucket`              |
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`.
+
+Alternatively, a YAML file that specifies the values for the parameters can be provided while installing the chart. For example,
+
+```bash
+$ helm install --name my-release -f values.yaml patroni-x.x.x.tgz
+```
+
+> **Tip**: You can use the default [values.yaml](values.yaml)
+
+## Cleanup
+
+In order to remove everything you created a simple `helm delete <release-name>` isn't enough (as of now), but you can do the following:
+
+```bash
+$ helm delete <release-name>
+$ kubectl delete petset,po,pvc,svc,secret -l release=<release-name>
```

Comment:
This won't work btw.  Lets check this bashfu into here:

```console
$ grace=$(kubectl get po web-0 --template '{{.spec.terminationGracePeriodSeconds}}')
$ kubectl delete petset,po -l app=nginx
$ sleep $grace
$ kubectl delete pvc -l app=nginx
```

Update nginx with your petset.  Also the code block should be "console" not "bash".